src/storage/src/table/test_relational_table.rs

```diff
@@ -549,9 +581,9 @@ async fn test_state_table_iter() {
     // this row exists in cell_based_table
     assert_eq!(
         Row(vec![
-            Some(1_i32.into()),
-            Some(11_i32.into()),
-            Some(111_i32.into())
+            Some(0_i32.into()),
+            Some(00_i32.into()),
+            Some(000_i32.into())
         ]),
```

Comment:
Why insert 0, 00, 000 to test the iter will call next if pk is in mem table (delete) and cell based table. I think if not insert (0, 00, 000), the iter get 3, 33, 333 can also test that